[Automation] javac compile fails for com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0

## javac compile on ubuntu-latest / JDK 25 / mode current-defaults

Failure kind: javac compile
OS: ubuntu-latest
JDK: 25
Native-image mode: current-defaults
Reproducer: `env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean compileTestJava -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/26742420600
Last 300 lines of the log:
```console
========================================================================================
 run-consecutive-tests input
========================================================================================
Coordinates: com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2
Raw versions JSON: [
  "2.22.0"
]
Parsed version count: 1
Parsed versions: 2.22.0
Timeout per Gradle invocation: 5m
========================================================================================
 Testing com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2:2.22.0
========================================================================================
========================================================================================
 com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2:2.22.0 stage native-image run
========================================================================================
Starting stage 'native-image run' for 2.22.0 at 2026-06-01T08:04:23Z
Command: env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean test -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
Starting a Gradle Daemon, 1 busy Daemon could not be reused, use --status for details
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean UP-TO-DATE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.21.2/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 6s
9 actionable tasks: 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :tckTest
====================
Testing library: com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2
Command: `/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/gradlew nativeTest`
Executing test...
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.21.2/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':nativeTestCompile'.
> Could not resolve all dependencies for configuration ':nativeImageTestClasspath'.
   > Could not resolve all dependencies for configuration ':nativeImageTestClasspath'.
      > Could not find com.fasterxml.jackson.core:jackson-core:2.22.0.
        Required by:
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson:jackson-bom:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.module:jackson-module-jakarta-xmlbind-annotations:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0 > com.fasterxml.jackson.core:jackson-databind:2.22.0

* Try:
> The project declares repositories, effectively ignoring the repositories you have declared in the settings.
   To determine how project repositories are declared, configure your build to fail on project repositories.
   For more information, please refer to https://docs.gradle.org/9.1.0/userguide/declaring_repositories.html#sub:fail_build_on_project_repositories in the Gradle documentation.
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 2s
8 actionable tasks: 8 up-to-date

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 11s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 2.22.0 at 2026-06-01T08:04:35Z with exit code 1 after 12s
❌ ERROR: 'native-image run' for 2.22.0 failed with exit code 1.
SOFT-FAIL[native-image run][2.22.0][env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean test -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2]
========================================================================================
 com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2:2.22.0 stage javac compile
========================================================================================
Starting stage 'javac compile' for 2.22.0 at 2026-06-01T08:04:35Z
Command: env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean compileTestJava -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.21.2/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 972ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava FAILED
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.21.2/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':compileTestJava'.
> Could not resolve all dependencies for configuration ':testCompileClasspath'.
   > Could not resolve all dependencies for configuration ':testCompileClasspath'.
      > Could not find com.fasterxml.jackson.core:jackson-core:2.22.0.
        Searched in the following locations:
          - file:/home/runner/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.22.0/jackson-core-2.22.0.pom
          - https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-core/2.22.0/jackson-core-2.22.0.pom
        Required by:
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson:jackson-bom:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.module:jackson-module-jakarta-xmlbind-annotations:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0 > com.fasterxml.jackson.core:jackson-databind:2.22.0

* Try:
> The project declares repositories, effectively ignoring the repositories you have declared in the settings.
   To determine how project repositories are declared, configure your build to fail on project repositories.
   For more information, please refer to https://docs.gradle.org/9.1.0/userguide/declaring_repositories.html#sub:fail_build_on_project_repositories in the Gradle documentation.
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.

> Run with --scan to generate a Build Scan (Powered by Develocity).Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.


> Get more help at https://help.gradle.org.
You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
8 actionable tasks: 8 up-to-date

BUILD FAILED in 931ms

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 3s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'javac compile' for 2.22.0 at 2026-06-01T08:04:38Z with exit code 1 after 3s
❌ ERROR: 'javac compile' for 2.22.0 failed with exit code 1.
FAILED[javac compile][2.22.0][env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean compileTestJava -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2]
```

## javac compile on ubuntu-latest / JDK latest-ea / mode current-defaults

Failure kind: javac compile
OS: ubuntu-latest
JDK: latest-ea
Native-image mode: current-defaults
Reproducer: `env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean compileTestJava -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/26742420600
Last 300 lines of the log:
```console
========================================================================================
 run-consecutive-tests input
========================================================================================
Coordinates: com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2
Raw versions JSON: [
  "2.22.0"
]
Parsed version count: 1
Parsed versions: 2.22.0
Timeout per Gradle invocation: 5m
========================================================================================
 Testing com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2:2.22.0
========================================================================================
========================================================================================
 com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2:2.22.0 stage native-image run
========================================================================================
Starting stage 'native-image run' for 2.22.0 at 2026-06-01T08:04:13Z
Command: env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean test -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
Starting a Gradle Daemon, 1 busy Daemon could not be reused, use --status for details
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean UP-TO-DATE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.21.2/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 6s
9 actionable tasks: 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :tckTest
====================
Testing library: com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2
Command: `/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/gradlew nativeTest`
Executing test...
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.21.2/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':nativeTestCompile'.
> Could not resolve all dependencies for configuration ':nativeImageTestClasspath'.
   > Could not resolve all dependencies for configuration ':nativeImageTestClasspath'.
      > Could not find com.fasterxml.jackson.core:jackson-core:2.22.0.
        Required by:
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson:jackson-bom:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.module:jackson-module-jakarta-xmlbind-annotations:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0 > com.fasterxml.jackson.core:jackson-databind:2.22.0

* Try:
> The project declares repositories, effectively ignoring the repositories you have declared in the settings.
   To determine how project repositories are declared, configure your build to fail on project repositories.
   For more information, please refer to https://docs.gradle.org/9.1.0/userguide/declaring_repositories.html#sub:fail_build_on_project_repositories in the Gradle documentation.
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

BUILD FAILED in 3s

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
8 actionable tasks: 8 up-to-date

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 11s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 2.22.0 at 2026-06-01T08:04:25Z with exit code 1 after 12s
❌ ERROR: 'native-image run' for 2.22.0 failed with exit code 1.
SOFT-FAIL[native-image run][2.22.0][env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean test -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2]
========================================================================================
 com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2:2.22.0 stage javac compile
========================================================================================
Starting stage 'javac compile' for 2.22.0 at 2026-06-01T08:04:25Z
Command: env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean compileTestJava -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.21.2/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 787ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava FAILED
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE


[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.21.2/build/reports/problems/problems-report.html
FAILURE: 
Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.
Build failed with an exception.


You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.
* What went wrong:

Could not determine the dependencies of task ':compileTestJava'.
For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
> Could not resolve all dependencies for configuration ':testCompileClasspath'.
8 actionable tasks: 8 up-to-date
   > Could not resolve all dependencies for configuration ':testCompileClasspath'.
      > Could not find com.fasterxml.jackson.core:jackson-core:2.22.0.
        Searched in the following locations:
          - file:/home/runner/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.22.0/jackson-core-2.22.0.pom
          - https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-core/2.22.0/jackson-core-2.22.0.pom
        Required by:
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson:jackson-bom:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.module:jackson-module-jakarta-xmlbind-annotations:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0 > com.fasterxml.jackson.core:jackson-databind:2.22.0

* Try:
> The project declares repositories, effectively ignoring the repositories you have declared in the settings.
   To determine how project repositories are declared, configure your build to fail on project repositories.
   For more information, please refer to https://docs.gradle.org/9.1.0/userguide/declaring_repositories.html#sub:fail_build_on_project_repositories in the Gradle documentation.
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

BUILD FAILED in 917ms

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 2s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'javac compile' for 2.22.0 at 2026-06-01T08:04:28Z with exit code 1 after 3s
❌ ERROR: 'javac compile' for 2.22.0 failed with exit code 1.
FAILED[javac compile][2.22.0][env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=current-defaults ./gradlew clean compileTestJava -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2]
```

## javac compile on ubuntu-latest / JDK latest-ea / mode future-defaults-all

Failure kind: javac compile
OS: ubuntu-latest
JDK: latest-ea
Native-image mode: future-defaults-all
Reproducer: `env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=future-defaults-all ./gradlew clean compileTestJava -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/26742420600
Last 300 lines of the log:
```console
========================================================================================
 run-consecutive-tests input
========================================================================================
Coordinates: com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2
Raw versions JSON: [
  "2.22.0"
]
Parsed version count: 1
Parsed versions: 2.22.0
Timeout per Gradle invocation: 5m
========================================================================================
 Testing com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2:2.22.0
========================================================================================
========================================================================================
 com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2:2.22.0 stage native-image run
========================================================================================
Starting stage 'native-image run' for 2.22.0 at 2026-06-01T08:04:17Z
Command: env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=future-defaults-all ./gradlew clean test -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
Starting a Gradle Daemon, 1 busy Daemon could not be reused, use --status for details
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean UP-TO-DATE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.21.2/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 6s
9 actionable tasks: 9 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :tckTest
====================
Testing library: com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2
Command: `/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/gradlew nativeTest`
Executing test...
-------
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.21.2/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':nativeTestCompile'.
> Could not resolve all dependencies for configuration ':nativeImageTestClasspath'.
   > Could not resolve all dependencies for configuration ':nativeImageTestClasspath'.
      > Could not find com.fasterxml.jackson.core:jackson-core:2.22.0.
        Required by:
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson:jackson-bom:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.module:jackson-module-jakarta-xmlbind-annotations:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0 > com.fasterxml.jackson.core:jackson-databind:2.22.0

* Try:
> The project declares repositories, effectively ignoring the repositories you have declared in the settings.
   To determine how project repositories are declared, configure your build to fail on project repositories.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
8 actionable tasks: 8 up-to-date
   For more information, please refer to https://docs.gradle.org/9.1.0/userguide/declaring_repositories.html#sub:fail_build_on_project_repositories in the Gradle documentation.
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

BUILD FAILED in 2s

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 11s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'native-image run' for 2.22.0 at 2026-06-01T08:04:28Z with exit code 1 after 11s
❌ ERROR: 'native-image run' for 2.22.0 failed with exit code 1.
SOFT-FAIL[native-image run][2.22.0][env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=future-defaults-all ./gradlew clean test -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2]
========================================================================================
 com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2:2.22.0 stage javac compile
========================================================================================
Starting stage 'javac compile' for 2.22.0 at 2026-06-01T08:04:28Z
Command: env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=future-defaults-all ./gradlew clean compileTestJava -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.21.2/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 880ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava FAILED
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.21.2/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Could not determine the dependencies of task ':compileTestJava'.
> Could not resolve all dependencies for configuration ':testCompileClasspath'.
   > Could not resolve all dependencies for configuration ':testCompileClasspath'.
      > Could not find com.fasterxml.jackson.core:jackson-core:2.22.0.
        Searched in the following locations:
          - file:/home/runner/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.22.0/jackson-core-2.22.0.pom
          - https://repo.maven.apache.org/maven2/com/fasterxml/jackson/core/jackson-core/2.22.0/jackson-core-2.22.0.pom
        Required by:
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson:jackson-bom:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.module:jackson-module-jakarta-xmlbind-annotations:2.22.0
            root project 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests' > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0 > com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-base:2.22.0 > com.fasterxml.jackson.core:jackson-databind:2.22.0

* Try:
> The project declares repositories, effectively ignoring the repositories you have declared in the settings.
   To determine how project repositories are declared, configure your build to fail on project repositories.
   For more information, please refer to https://docs.gradle.org/9.1.0/userguide/declaring_repositories.html#sub:fail_build_on_project_repositories in the Gradle documentation.
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

BUILD FAILED in 970ms

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
8 actionable tasks: 8 up-to-date

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 2s
11 actionable tasks: 3 executed, 8 up-to-date
Finished stage 'javac compile' for 2.22.0 at 2026-06-01T08:04:31Z with exit code 1 after 3s
❌ ERROR: 'javac compile' for 2.22.0 failed with exit code 1.
FAILED[javac compile][2.22.0][env GVM_TCK_LV=2.22.0 GVM_TCK_NATIVE_IMAGE_MODE=future-defaults-all ./gradlew clean compileTestJava -Pcoordinates=com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2]
```

--- a/metadata/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/reachability-metadata.json
+++ b/metadata/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/reachability-metadata.json
@@ -1,108 +1,58 @@
 {
-  "reflection": [
+  "reflection" : [
     {
-      "condition": {
-        "typeReached": "com.fasterxml.jackson.jakarta.rs.yaml.YAMLMapperConfigurator"
+      "condition" : {
+        "typeReached" : "com.fasterxml.jackson.jakarta.rs.yaml.YAMLMapperConfigurator"
       },
-      "type": "com.fasterxml.jackson.databind.ext.Java7SupportImpl",
-      "methods": [
+      "type" : "com.fasterxml.jackson.databind.ext.Java7SupportImpl",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "com.fasterxml.jackson.jakarta.rs.yaml.YAMLMapperConfigurator"
+      "condition" : {
+        "typeReached" : "com.fasterxml.jackson.jakarta.rs.yaml.YAMLMapperConfigurator"
       },
-      "type": "com.fasterxml.jackson.module.jakarta.xmlbind.JakartaXmlBindAnnotationIntrospector",
-      "methods": [
+      "type" : "com.fasterxml.jackson.module.jakarta.xmlbind.JakartaXmlBindAnnotationIntrospector",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "com.fasterxml.jackson.jakarta.rs.yaml.YAMLMapperConfigurator"
+      "condition" : {
+        "typeReached" : "com.fasterxml.jackson.jakarta.rs.yaml.YAMLMapperConfigurator"
       },
-      "type": "com.fasterxml.jackson.module.jakarta.xmlbind.deser.DataHandlerDeserializer",
-      "methods": [
+      "type" : "com.fasterxml.jackson.module.jakarta.xmlbind.deser.DataHandlerDeserializer",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "com.fasterxml.jackson.jakarta.rs.yaml.YAMLMapperConfigurator"
+      "condition" : {
+        "typeReached" : "com.fasterxml.jackson.jakarta.rs.yaml.YAMLMapperConfigurator"
       },
-      "type": "com.fasterxml.jackson.module.jakarta.xmlbind.ser.DataHandlerSerializer",
-      "methods": [
+      "type" : "com.fasterxml.jackson.module.jakarta.xmlbind.ser.DataHandlerSerializer",
+      "methods" : [
         {
-          "name": "<init>",
-          "parameterTypes": []
+          "name" : "<init>",
+          "parameterTypes" : [ ]
         }
       ]
     },
     {
-      "condition": {
-        "typeReached": "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest"
+      "condition" : {
+        "typeReached" : "com.fasterxml.jackson.jakarta.rs.yaml.YAMLMapperConfigurator"
       },
-      "type": "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest$JakartaXmlBindBean",
-      "methods": [
-        {
-          "name": "getValue",
-          "parameterTypes": []
-        }
-      ]
-    },
-    {
-      "condition": {
-        "typeReached": "com.fasterxml.jackson.jakarta.rs.yaml.YAMLMapperConfigurator"
-      },
-      "type": "java.lang.Class"
-    },
-    {
-      "condition": {
-        "typeReached": "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest"
-      },
-      "type": "java.lang.Class",
-      "methods": [
-        {
-          "name": "isRecord",
-          "parameterTypes": []
-        }
-      ]
-    },
-    {
-      "condition": {
-        "typeReached": "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest"
-      },
-      "type": "java.sql.Date"
-    },
-    {
-      "condition": {
-        "typeReached": "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest"
-      },
-      "type": "java.sql.Timestamp"
-    }
-  ],
-  "resources": [
-    {
-      "condition": {
-        "typeReached": "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest"
-      },
-      "glob": "META-INF/services/org.assertj.core.configuration.Configuration"
-    },
-    {
-      "condition": {
-        "typeReached": "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest"
-      },
-      "glob": "META-INF/services/org.assertj.core.presentation.Representation"
+      "type" : "java.lang.Class"
     }
   ]
 }

--- a/metadata/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/reachability-metadata.json
+++ b/metadata/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/reachability-metadata.json
@@ -1,0 +1,108 @@
+{
+  "reflection": [
+    {
+      "condition": {
+        "typeReached": "com.fasterxml.jackson.jakarta.rs.yaml.YAMLMapperConfigurator"
+      },
+      "type": "com.fasterxml.jackson.databind.ext.Java7SupportImpl",
+      "methods": [
+        {
+          "name": "<init>",
+          "parameterTypes": []
+        }
+      ]
+    },
+    {
+      "condition": {
+        "typeReached": "com.fasterxml.jackson.jakarta.rs.yaml.YAMLMapperConfigurator"
+      },
+      "type": "com.fasterxml.jackson.module.jakarta.xmlbind.JakartaXmlBindAnnotationIntrospector",
+      "methods": [
+        {
+          "name": "<init>",
+          "parameterTypes": []
+        }
+      ]
+    },
+    {
+      "condition": {
+        "typeReached": "com.fasterxml.jackson.jakarta.rs.yaml.YAMLMapperConfigurator"
+      },
+      "type": "com.fasterxml.jackson.module.jakarta.xmlbind.deser.DataHandlerDeserializer",
+      "methods": [
+        {
+          "name": "<init>",
+          "parameterTypes": []
+        }
+      ]
+    },
+    {
+      "condition": {
+        "typeReached": "com.fasterxml.jackson.jakarta.rs.yaml.YAMLMapperConfigurator"
+      },
+      "type": "com.fasterxml.jackson.module.jakarta.xmlbind.ser.DataHandlerSerializer",
+      "methods": [
+        {
+          "name": "<init>",
+          "parameterTypes": []
+        }
+      ]
+    },
+    {
+      "condition": {
+        "typeReached": "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest"
+      },
+      "type": "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest$JakartaXmlBindBean",
+      "methods": [
+        {
+          "name": "getValue",
+          "parameterTypes": []
+        }
+      ]
+    },
+    {
+      "condition": {
+        "typeReached": "com.fasterxml.jackson.jakarta.rs.yaml.YAMLMapperConfigurator"
+      },
+      "type": "java.lang.Class"
+    },
+    {
+      "condition": {
+        "typeReached": "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest"
+      },
+      "type": "java.lang.Class",
+      "methods": [
+        {
+          "name": "isRecord",
+          "parameterTypes": []
+        }
+      ]
+    },
+    {
+      "condition": {
+        "typeReached": "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest"
+      },
+      "type": "java.sql.Date"
+    },
+    {
+      "condition": {
+        "typeReached": "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest"
+      },
+      "type": "java.sql.Timestamp"
+    }
+  ],
+  "resources": [
+    {
+      "condition": {
+        "typeReached": "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest"
+      },
+      "glob": "META-INF/services/org.assertj.core.configuration.Configuration"
+    },
+    {
+      "condition": {
+        "typeReached": "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest"
+      },
+      "glob": "META-INF/services/org.assertj.core.presentation.Representation"
+    }
+  ]
+}

--- a/metadata/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/index.json
+++ b/metadata/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/index.json
@@ -2,6 +2,11 @@
   {
     "latest" : true,
     "metadata-version" : "2.22.0",
+    "source-code-url" : "https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jakarta/rs/jackson-jakarta-rs-yaml-provider/$version$/jackson-jakarta-rs-yaml-provider-$version$-sources.jar",
+    "repository-url" : "https://github.com/FasterXML/jackson-jakarta-rs-providers",
+    "test-code-url" : "https://github.com/FasterXML/jackson-jakarta-rs-providers/tree/jackson-jakarta-rs-providers-$version$/yaml/src/test",
+    "documentation-url" : "https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jakarta/rs/jackson-jakarta-rs-yaml-provider/$version$/jackson-jakarta-rs-yaml-provider-$version$-javadoc.jar",
+    "description" : "Jackson Jakarta-RS YAML Provider integrates Jackson data binding with Jakarta RESTful Web Services implementations to read and write YAML request and response bodies. It supplies YAML MessageBodyReader and MessageBodyWriter providers, including JAXB-annotation-aware support, for frameworks such as Jersey and RESTEasy.",
     "tested-versions" : [
       "2.22.0"
     ],

--- a/metadata/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/index.json
+++ b/metadata/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/index.json
@@ -1,6 +1,15 @@
 [
   {
     "latest" : true,
+    "metadata-version" : "2.22.0",
+    "tested-versions" : [
+      "2.22.0"
+    ],
+    "allowed-packages" : [
+      "com.fasterxml.jackson.jakarta.rs.yaml"
+    ]
+  },
+  {
     "metadata-version" : "2.21.2",
     "source-code-url" : "https://repo.maven.apache.org/maven2/com/fasterxml/jackson/jakarta/rs/jackson-jakarta-rs-yaml-provider/$version$/jackson-jakarta-rs-yaml-provider-$version$-sources.jar",
     "repository-url" : "https://github.com/FasterXML/jackson-jakarta-rs-providers",

--- a/stats/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/execution-metrics.json
+++ b/stats/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/execution-metrics.json
@@ -1,0 +1,100 @@
+{
+  "fix_javac_fail:2026-06-05": {
+    "timestamp": "2026-06-05T14:07:18.671234Z",
+    "library": "com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0",
+    "starting_commit": "628ba691ad7ac077e2d84e2287185b14fe557dfa",
+    "ending_commit": "dcf79486f41618e89b2136d5bf6c88a7a9895180",
+    "previous_library": "com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.21.2",
+    "strategy_name": "javac_iterative_with_coverage_sources_pi_gpt-5.5",
+    "agent": "pi",
+    "model": "oca/gpt-5.5",
+    "status": "success",
+    "stats": {
+      "version": "2.22.0",
+      "dynamicAccess": {
+        "breakdown": {
+          "reflection": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 2,
+            "totalCalls": 2
+          }
+        },
+        "coverageRatio": 1.0,
+        "coveredCalls": 2,
+        "totalCalls": 2
+      },
+      "libraryCoverage": {
+        "instruction": {
+          "covered": 106,
+          "missed": 251,
+          "ratio": 0.296919,
+          "total": 357
+        },
+        "line": {
+          "covered": 23,
+          "missed": 64,
+          "ratio": 0.264368,
+          "total": 87
+        },
+        "method": {
+          "covered": 4,
+          "missed": 26,
+          "ratio": 0.133333,
+          "total": 30
+        }
+      }
+    },
+    "previous_library_stats": {
+      "version": "2.21.2",
+      "dynamicAccess": {
+        "breakdown": {
+          "reflection": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 2,
+            "totalCalls": 2
+          }
+        },
+        "coverageRatio": 1.0,
+        "coveredCalls": 2,
+        "totalCalls": 2
+      },
+      "libraryCoverage": {
+        "instruction": {
+          "covered": 106,
+          "missed": 251,
+          "ratio": 0.296919,
+          "total": 357
+        },
+        "line": {
+          "covered": 23,
+          "missed": 64,
+          "ratio": 0.264368,
+          "total": 87
+        },
+        "method": {
+          "covered": 4,
+          "missed": 26,
+          "ratio": 0.133333,
+          "total": 30
+        }
+      }
+    },
+    "metrics": {
+      "input_tokens_used": 136664,
+      "cached_input_tokens_used": 1076224,
+      "output_tokens_used": 15583,
+      "iterations": 3,
+      "cost_usd": 1.0056,
+      "tested_library_loc": 23,
+      "metadata_entries": 9,
+      "test_only_metadata_entries": 8,
+      "previous_library_metadata_entries": 6,
+      "code_coverage_percent": 26.44,
+      "previous_library_coverage_percent": 26.44
+    },
+    "artifacts": {
+      "test_file": "tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/src/test/java/com_fasterxml_jackson_jakarta_rs/jackson_jakarta_rs_yaml_provider/YAMLMapperConfiguratorTest.java",
+      "metadata_file": "metadata/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/reachability-metadata.json"
+    }
+  }
+}

--- a/stats/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/stats.json
+++ b/stats/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/stats.json
@@ -1,0 +1,37 @@
+{
+  "versions" : [ {
+    "version" : "2.22.0",
+    "dynamicAccess" : {
+      "breakdown" : {
+        "reflection" : {
+          "coverageRatio" : 1.0,
+          "coveredCalls" : 2,
+          "totalCalls" : 2
+        }
+      },
+      "coverageRatio" : 1.0,
+      "coveredCalls" : 2,
+      "totalCalls" : 2
+    },
+    "libraryCoverage" : {
+      "instruction" : {
+        "covered" : 106,
+        "missed" : 251,
+        "ratio" : 0.296919,
+        "total" : 357
+      },
+      "line" : {
+        "covered" : 23,
+        "missed" : 64,
+        "ratio" : 0.264368,
+        "total" : 87
+      },
+      "method" : {
+        "covered" : 4,
+        "missed" : 26,
+        "ratio" : 0.133333,
+        "total" : 30
+      }
+    }
+  } ]
+}

--- a/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/.gitignore
+++ b/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/.gitignore
@@ -1,0 +1,4 @@
+gradlew.bat
+gradlew
+gradle/
+build/

--- a/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/build.gradle
+++ b/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/build.gradle
@@ -1,0 +1,27 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+plugins {
+    id "org.graalvm.internal.tck"
+}
+
+String libraryVersion = tck.testedLibraryVersion.get()
+
+dependencies {
+    testImplementation "com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:$libraryVersion"
+    testImplementation 'org.assertj:assertj-core:3.22.0'
+}
+
+graalvmNative {
+    agent {
+        defaultMode = "conditional"
+        modes {
+            conditional {
+                userCodeFilterPath = "user-code-filter.json"
+            }
+        }
+    }
+}

--- a/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/gradle.properties
+++ b/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/gradle.properties
@@ -1,0 +1,6 @@
+# Optional explicit tested library coordinates (format: group:artifact:version).
+# If omitted, fallback resolution in org.graalvm.internal.tck.gradle still uses
+# environment variable GVM_TCK_LC and then this property.
+library.coordinates = com.fasterxml.jackson.jakarta.rs:jackson-jakarta-rs-yaml-provider:2.22.0
+library.version = 2.22.0
+metadata.dir = com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/

--- a/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/settings.gradle
+++ b/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/settings.gradle
@@ -1,0 +1,20 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+pluginManagement {
+    def tckPath = Objects.requireNonNullElse(
+            System.getenv("GVM_TCK_TCKDIR"),
+            "../../../../tck-build-logic"
+    )
+    includeBuild(tckPath)
+    repositories { mavenLocal(); gradlePluginPortal() }
+}
+
+plugins {
+    id "org.graalvm.internal.tck-settings" version "1.0.0-SNAPSHOT"
+}
+
+rootProject.name = 'com.fasterxml.jackson.jakarta.rs.jackson-jakarta-rs-yaml-provider_tests'

--- a/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/src/test/java/com_fasterxml_jackson_jakarta_rs/jackson_jakarta_rs_yaml_provider/YAMLMapperConfiguratorTest.java
+++ b/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/src/test/java/com_fasterxml_jackson_jakarta_rs/jackson_jakarta_rs_yaml_provider/YAMLMapperConfiguratorTest.java
@@ -1,0 +1,47 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.fasterxml.jackson.databind.AnnotationIntrospector;
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
+import com.fasterxml.jackson.jakarta.rs.cfg.Annotations;
+import com.fasterxml.jackson.jakarta.rs.yaml.YAMLMapperConfigurator;
+import com.fasterxml.jackson.module.jakarta.xmlbind.JakartaXmlBindAnnotationIntrospector;
+import jakarta.xml.bind.annotation.XmlElement;
+import org.junit.jupiter.api.Test;
+
+public class YAMLMapperConfiguratorTest {
+    @Test
+    void defaultMapperResolvesJakartaXmlBindAnnotationIntrospector() throws Exception {
+        YAMLMapperConfigurator configurator = new YAMLMapperConfigurator(
+                null,
+                new Annotations[] {Annotations.JAKARTA_XML_BIND });
+
+        YAMLMapper mapper = configurator.getDefaultMapper();
+
+        AnnotationIntrospector introspector = mapper.getSerializationConfig().getAnnotationIntrospector();
+        assertThat(introspector).isInstanceOf(JakartaXmlBindAnnotationIntrospector.class);
+
+        String yaml = mapper.writeValueAsString(new JakartaXmlBindBean("configured"));
+        assertThat(yaml).contains("xmlName:", "configured");
+    }
+
+    public static final class JakartaXmlBindBean {
+        private final String value;
+
+        public JakartaXmlBindBean(String value) {
+            this.value = value;
+        }
+
+        @XmlElement(name = "xmlName")
+        public String getValue() {
+            return value;
+        }
+    }
+}

--- a/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/src/test/java/com_fasterxml_jackson_jakarta_rs/jackson_jakarta_rs_yaml_provider/YAMLMapperConfiguratorTest.java
+++ b/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/src/test/java/com_fasterxml_jackson_jakarta_rs/jackson_jakarta_rs_yaml_provider/YAMLMapperConfiguratorTest.java
@@ -25,7 +25,8 @@ public class YAMLMapperConfiguratorTest {
 
         YAMLMapper mapper = configurator.getDefaultMapper();
 
-        AnnotationIntrospector introspector = mapper.getSerializationConfig().getAnnotationIntrospector();
+        AnnotationIntrospector introspector = mapper.getSerializationConfig()
+                .getAnnotationIntrospector();
         assertThat(introspector).isInstanceOf(JakartaXmlBindAnnotationIntrospector.class);
 
         String yaml = mapper.writeValueAsString(new JakartaXmlBindBean("configured"));

--- a/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/src/test/resources/META-INF/native-image/reachability-metadata.json
+++ b/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/src/test/resources/META-INF/native-image/reachability-metadata.json
@@ -1,0 +1,54 @@
+{
+  "reflection" : [
+    {
+      "condition" : {
+        "typeReached" : "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest"
+      },
+      "type" : "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest$JakartaXmlBindBean",
+      "methods" : [
+        {
+          "name" : "getValue",
+          "parameterTypes" : [ ]
+        }
+      ]
+    },
+    {
+      "condition" : {
+        "typeReached" : "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest"
+      },
+      "type" : "java.lang.Class",
+      "methods" : [
+        {
+          "name" : "isRecord",
+          "parameterTypes" : [ ]
+        }
+      ]
+    },
+    {
+      "condition" : {
+        "typeReached" : "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest"
+      },
+      "type" : "java.sql.Date"
+    },
+    {
+      "condition" : {
+        "typeReached" : "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest"
+      },
+      "type" : "java.sql.Timestamp"
+    }
+  ],
+  "resources" : [
+    {
+      "condition" : {
+        "typeReached" : "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest"
+      },
+      "glob" : "META-INF/services/org.assertj.core.configuration.Configuration"
+    },
+    {
+      "condition" : {
+        "typeReached" : "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.YAMLMapperConfiguratorTest"
+      },
+      "glob" : "META-INF/services/org.assertj.core.presentation.Representation"
+    }
+  ]
+}

--- a/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/user-code-filter.json
+++ b/tests/src/com.fasterxml.jackson.jakarta.rs/jackson-jakarta-rs-yaml-provider/2.22.0/user-code-filter.json
@@ -1,0 +1,13 @@
+{
+  "rules" : [
+    {
+      "excludeClasses" : "**"
+    },
+    {
+      "includeClasses" : "com.fasterxml.jackson.jakarta.rs.yaml.**"
+    },
+    {
+      "includeClasses" : "com_fasterxml_jackson_jakarta_rs.jackson_jakarta_rs_yaml_provider.**"
+    }
+  ]
+}

